Makefile

```diff
@@ -48,13 +65,20 @@ test: .env ## Runs unit tests with py and code checks against staged changes.
 .PHONY: integration
 integration: .env ## Runs postgres integration tests with py-integration
 	@\
-	$(DOCKER_CMD) tox -e py-integration -- -nauto
+	$(COND_CI_FLAGS) $(DOCKER_CMD) tox -e py-integration -- -nauto
 
 .PHONY: integration-fail-fast
 integration-fail-fast: .env ## Runs postgres integration tests with py-integration in "fail fast" mode.
 	@\
 	$(DOCKER_CMD) tox -e py-integration -- -x -nauto
 
+.PHONY: structured-log-test
+structured-log-test: clean
```

Comment:
I'm inclined to name this "interop" instead. I know structured logs are the only tests in there now, but in theory interop is a layer of abstraction above integration tests. unit (blackbox=function) < integration (blackbox=function+warehouse) < interop (blackbox=dbt+warehouse).

Reply:
Sure! Can do